test/e2e/multihoming.go

```diff
@@ -24,11 +24,13 @@ import (
 
 var _ = Describe("Multi Homing", func() {
 	const (
-		podName                    = "tinypod"
-		secondaryNetworkCIDR       = "10.128.0.0/16"
-		secondaryNetworkName       = "tenant-blue"
-		secondaryFlatL2IgnoreCIDR  = "10.128.0.0/29"
-		secondaryFlatL2NetworkCIDR = "10.128.0.0/24"
+		podName                      = "tinypod"
+		secondaryNetworkCIDR         = "10.128.0.0/16"
+		secondaryNetworkName         = "tenant-blue"
+		secondaryFlatL2IgnoreCIDR    = "10.128.0.0/29"
+		secondaryFlatL2NetworkCIDR   = "10.128.0.0/24"
+		secondaryLocalnetIgnoreCIDR  = "60.128.0.0/29"
+		secondaryLocalnetNetworkCIDR = "60.128.0.0/24"
```

Comment:
do we really need a different CIDR for this type of networks ?

Reply:
we don't. but I thought we need to allow it. If one day we are able to somehow make use of underlay networking, we may need specific CIDR/VLANID combination.

Reply:
IMO, we should add these once that day comes then.

I won't insist further.